testing/python/issue/test_tilelang_issue_2963.py

```
@@ -0,0 +1,318 @@
+def test_multiple_packed_store_sites_are_rejected_when_ownership_disagrees():
+    n = 132
+
+    @T.prim_func
+    def kernel(A: T.Tensor((n,), "int4"), B: T.Tensor((n,), "int4")):
+        with T.Kernel(1, threads=64):
+            for i in T.Parallel(64):
+                B[2 * i] = A[2 * i]
+                shifted = 2 * i + 3
+                B[shifted] = A[shifted]
+
+    with pytest.raises(Exception, match="no available layout found|no legal vector width"):
```

Ruff (0.16.2)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)